libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/ParentChildrenSyncDownTarget.java

```diff
@@ -28,9 +28,8 @@
 
 import android.text.TextUtils;
 
-import com.salesforce.androidsdk.smartstore.store.QuerySpec;
-import com.salesforce.androidsdk.smartstore.store.SmartStore;
 import com.salesforce.androidsdk.smartsync.manager.SyncManager;
+import com.salesforce.androidsdk.smartsync.target.ParentChildrenSyncTargetHelper.RelationshipType;
```

Comment:
Code shared by ParentChildrenSyncDownTarget and ParentChildrenSyncUpTarget moved to ParentChildrenSyncTargetHelper